student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  attattr_accessor :@classroom
+  def initialize(classroom, name, age, parent_permission)
+    @classroom = classroom
+    super(name, age, parent_permission)
+  end
```

Comment:
- [ ] in this constructor there should also be a default value for name and parent_permission.